app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java

```diff
@@ -1067,5 +1069,49 @@ protected void onPostExecute(Void result) {
                 SnackBarHandler.show(parent, getString(R.string.error_on_tumblr));
         }
     }
+    public File saveBitmapToFile(File file){
```

Comment:
@Subhankar29 I was talking about this function.
I don't understand what is the need of this. You created the file and used it just to show the preview. You could've achieved the same by adding .resize(w,h) to glide method or use universal image loader. If your intention is something other then please mention